### PR TITLE
style(suite): Redesign AccountsMenu

### PR DESCRIPTION
**Pull request by jvaclavik**
## Related Issue

Resolve https://github.com/trezor/trezor-suite/issues/9912

## Screenshots:

image

image

mobile
image

shadow
image

image

### CODE REVIEW

File: packages/components/src/components/Dropdown/getAdjustedCoords.ts [hunk 1]
```diff
@@ -3,49 +3,92 @@ import { MenuAlignment } from './Menu';
...
         return;
     }
 
     let { x, y } = coords;
-    const { width, height } = menuDimentions;
-    const OFFSET = 4;
+    const { width, height } = menuDimensions;
+    const MENU_DEFAULT_OFFSET = 4;
 
     switch (alignMenu) {
```
Comment (jvaclavik) [inline packages/components/src/components/Dropdown/getAdjustedCoords.ts:29]: We might use two props instead of one (something like `positionFromMenu` and `dropdownAlignment`

File: packages/components/src/components/Badge/Badge.tsx [hunk 2]
```diff
@@ -59,14 +59,24 @@ const getIconColor = (variant: BadgeVariant, isDisabled: boolean | undefined): C
...
 const Container = styled.button<BadgeContainerProps>`
     display: flex;
     align-items: center;
     gap: ${spacingsPx.xxs};
-    padding: ${({ size }) =>
-        size === 'small' ? `0 ${spacingsPx.xs}` : `${spacingsPx.xxxs} ${spacingsPx.xs}`};
+    padding: ${({ size }) => getPadding(size)};
     border-radius: ${borders.radii.full};
     border: 1px solid transparent;
     background: ${({ variant, theme }) => getBackgroundColor(variant, theme)};
```
Comment (jvaclavik) [inline packages/components/src/components/Badge/Badge.tsx:82]: we should also update background color to not have opacity (it looks a bit weird when you have badge on the top of the coin logo)

File: packages/components/src/components/Dropdown/Dropdown.tsx [hunk 3]
```diff
@@ -87,7 +87,9 @@ export const Dropdown = forwardRef(
             isDisabled,
             renderOnClickPosition,
             addon,
-            alignMenu = 'left',
+            alignMenu = 'bottom-left',
```
Comment (jvaclavik) [inline packages/components/src/components/Dropdown/Dropdown.tsx:90]: first is position from the toggle icon, second alignment on the dropdown itself related to this icon

image

image

image

etc.

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx [hunk 4]
```diff
@@ -88,18 +98,57 @@ const ExpandedMobileWrapper = styled.div`
...
+    overflow: auto;
+    position: relative;
+
+    ::before {
+        content: '';
+        width: 100%;
+        height: 45px;
+        background: linear-gradient(
+            ${({ theme }) => theme.backgroundSurfaceElevationNegative},
+            rgba(0 0 0 / 0%)
```
Comment (jvaclavik) [inline packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx]: This is really weird notation, but it doesn't break linter like other ones 🤷‍♂️

File: packages/components/src/components/form/Input/Input.tsx [hunk 5]
```diff
@@ -91,6 +91,8 @@ export interface InputProps extends Omit<InputHTMLAttributes<HTMLInputElement>,
     inputState?: InputState;
     innerAddonAlign?: innerAddonAlignment;
     hasBottomPadding?: boolean;
+    leftAddonWidth?: number;
```
Comment (dahaca) [inline packages/components/src/components/form/Input/Input.tsx]: I would not do it through props. As a rule of thumb – I believe style-related props are never good.

* they bloat component API, making it harder to read and understand what _functionality_ there is, not what it looks like
* essentially mixing 2 domains can lead to some styles being in CSS and some in props – not that sweet

I would even not be against static `classNames` for the secondary components, or styled components added to the component [like in the Modal](https://github.com/trezor/trezor-suite/blob/0126f4054d50ac1b15a1d9013794b1470be783d3/packages/suite/src/components/suite/modals/Modal/Modal.tsx#L18)

In this case, you do not need props at all – `styled(Input)` would work, as `className` is passed to `input` as a part of `...rest`

File: packages/components/src/components/assets/CoinLogo/CoinLogo.tsx [hunk 6]
```diff
@@ -131,27 +128,21 @@ const Logo = ({
     </SvgWrapper>
 );
 
-const CoinLogo = ({
+export const CoinLogoWithShareIndicator = ({
```
Comment (dahaca) [inline packages/components/src/components/assets/CoinLogo/CoinLogo.tsx]: split into multiple ifles?

File: packages/suite/src/components/suite/AccountLabel.tsx [hunk 7]
```diff
@@ -3,6 +3,10 @@ import { getTitleForNetwork, getTitleForCoinjoinAccount } from '@suite-common/wa
 import { Translation } from 'src/components/suite';
 import { Account } from 'src/types/wallet';
 
+const EllipsisContainer = styled.span`
```
Comment (dahaca) [inline packages/suite/src/components/suite/AccountLabel.tsx]: nit: I believe this could be a separate component, with a tooltip containing the full name appearing on long hover. I would at least add the tooltip here and care about abstractions later.

File: packages/suite/src/components/suite/Preloader/SuiteLayout/Sidebar/Sidebar.tsx [hunk 8]
```diff
@@ -10,19 +10,25 @@ const Container = styled.nav`
     display: flex;
     flex-direction: column;
     flex: 0 0 auto;
-    gap: ${spacingsPx.xs};
     width: 240px;
     height: 100%;
-    padding: ${spacingsPx.xs};
```
Comment (dahaca) [inline packages/suite/src/components/suite/Preloader/SuiteLayout/Sidebar/Sidebar.tsx:16]: why? I moved all the padding here so that they are not stated multiple times. Also it breaks the `QuickAction` style :(

image

File: packages/suite/src/components/suite/Preloader/SuiteLayout/Sidebar/Sidebar.tsx [hunk 9]
```diff
@@ -10,19 +10,25 @@ const Container = styled.nav`
...
     background: ${({ theme }) => theme.backgroundSurfaceElevationNegative};
     border-right: 1px solid ${({ theme }) => theme.borderOnElevation0};
     overflow: auto;
 `;
+const Margin = styled.div`
```
Comment (dahaca) [inline packages/suite/src/components/suite/Preloader/SuiteLayout/Sidebar/Sidebar.tsx]: Not sure why is this needed either, maybe i am missing something?

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/CoinsFilter.tsx [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+        ${StyledCoinLogo} {
+            filter: none;
+        }
+    }
+`;
+
+const OuterCircle = styled.div<{ isSelected?: boolean; isMobile?: boolean }>`
+    height: 24px;
+    border-radius: 50%;
+    border: 4px solid
```
Comment (dahaca) [inline packages/suite/src/components/wallet/WalletLayout/AccountsMenu/CoinsFilter.tsx]: Make it 2px, please 🙏  And I like how the icons become colored on the filter hover! How about keeping the selected icon hovered?
image

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/CoinsFilter.tsx [hunk 11]
```diff
@@ -0,0 +1,85 @@
...
+    transition: all 0.3;
+    cursor: pointer;
+
+    :hover {
+        border: 4px solid ${({ theme }) => theme.backgroundSecondaryDefault};
+    }
+`;
+
+interface CoinsFilterProps {
+    isMobile?: boolean;
```
Comment (dahaca) [inline packages/suite/src/components/wallet/WalletLayout/AccountsMenu/CoinsFilter.tsx]: not used anywhere

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/CoinsFilter.tsx [hunk 12]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+export const CoinsFilter = ({ isMobile }: CoinsFilterProps) => {
```
Comment (dahaca) [inline packages/suite/src/components/wallet/WalletLayout/AccountsMenu/CoinsFilter.tsx]: completely missing on mobile, I think it would fit without any changes :) 
image

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/CoinsFilter.tsx [hunk 13]
```diff
@@ -0,0 +1,85 @@
...
+    const { coinFilter, setCoinFilter } = useAccountSearch();
+    const enabledNetworks = useSelector(state => state.wallet.settings.enabledNetworks);
+    const device = useSelector(selectDevice);
+
+    const unavailableCapabilities = device?.unavailableCapabilities ?? {};
+    const supportedNetworks = enabledNetworks.filter(symbol => !unavailableCapabilities[symbol]);
+
+    const showCoinFilter = supportedNetworks.length > 1;
+
```
Comment (dahaca) [inline packages/suite/src/components/wallet/WalletLayout/AccountsMenu/CoinsFilter.tsx:51]:
```
 if (!showCoinFilter) {
        return null;
}
```

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/CoinsFilter.tsx (hunk 12 above)
Comment (dahaca) [inline packages/suite/src/components/wallet/WalletLayout/AccountsMenu/CoinsFilter.tsx]: Filters are not fixed 
image

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx [hunk 14]
```diff
@@ -88,18 +90,57 @@ const ExpandedMobileWrapper = styled.div`
...
+
+    ::after {
+        content: '';
+        width: 100%;
+        height: 45px;
+        background: linear-gradient(
+            rgba(0 0 0 0%),
+            ${({ theme }) => theme.backgroundSurfaceElevationNegative}
+        );
+        z-index: 1;
```
Comment (dahaca) [inline packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx]: nit: try removing all z-indicies in all AcocuntMenu compoents, it all seems to work without them :)

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx [hunk 15]
```diff
@@ -88,18 +90,57 @@ const ExpandedMobileWrapper = styled.div`
...
+    flex: 1;
+    z-index: ${zIndices.expandableNavigationHeader};
+    width: 100%;
+    overflow: auto;
+    position: relative;
+
+    ::before {
+        content: '';
+        width: 100%;
+        height: 45px;
```
Comment (dahaca) [inline packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx]: nit: some styled can be abstracted into a shared style for clarity.

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx [hunk 16]
```diff
@@ -312,24 +350,18 @@ export const AccountsMenu = ({ isMenuInline }: AccountsMenuProps) => {
...
-                            </LoadingContent>
-                        </Heading>
-                        <AddAccountButton
-                            data-test="@account-menu/add-account"
-                            device={device}
-                            noButtonLabel
-                        />
-                    </Row>
+            <MenuHeader>
+                <Row>
```
Comment (dahaca) [inline packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx:113]: nit: don't think `Row` is needed here

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountSearchBox.tsx [hunk 17]
```diff
@@ -1,37 +1,12 @@
...
-    cursor: pointer;
-
-    margin-bottom: 8px;
-    margin-right: ${props => (props.isMobile ? '12px' : '4px')};
-`;
-
 const InputWrapper = styled.div<{ showCoinFilter: boolean }>`
+    flex: 1;
     ${props =>
         !props.showCoinFilter &&
```
Comment (dahaca) [inline packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountSearchBox.tsx]: this is not needed now + breaks the layout if left like that.

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountItem.tsx [hunk 18]
```diff
@@ -25,29 +25,36 @@ interface WrapperProps {
...
+    display: flex;
+    justify-content: space-between;
 
     & + & {
         margin-top: ${spacingsPx.xxs};
     }
 
     :hover {
+        position: relative;
+        z-index: 2;
```
Comment (dahaca) [inline packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountItem.tsx:38]: this is to elevate it above the shadows?

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountGroup.tsx [hunk 19]
```diff
@@ -82,29 +86,32 @@ export const AccountGroup = forwardRef((props: AccountGroupProps, _ref: Ref<HTML
...
     // Group needs to be wrapped into container (div)
     return (
-        <div ref={wrapperRef}>
+        <Container ref={wrapperRef}>
             <HeaderWrapper>
                 <Header
                     isOpen={isOpen}
                     onClick={!props.keepOpen ? onClick : undefined}
                     data-test={`@account-menu/${props.type}`}
+                    isVisible={props.type !== 'normal'}
```
Comment (dahaca) [inline packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountGroup.tsx]: there is a label config above on [line 52](https://github.com/trezor/trezor-suite/blob/8264cccf73551035b3ccf63ec1477cd842fbe940/packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountGroup.tsx#L52), would probably make sense to make it an object, remove the `normal` heading key from there and not render `Header` when there is no heading in the config?

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountGroup.tsx [hunk 20]
```diff
@@ -82,29 +86,32 @@ export const AccountGroup = forwardRef((props: AccountGroupProps, _ref: Ref<HTML
...
                 >
+                    <ChevronContainer>
+                        {!props.keepOpen && (
+                            <ChevronIcon
+                                data-test="@account-menu/arrow"
+                                canAnimate={animatedIcon}
+                                isActive={isOpen}
+                                size={16}
+                                color={theme.TYPE_LIGHT_GREY}
+                                icon="ARROW_DOWN"
```
Comment (dahaca) [inline packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountGroup.tsx]: old color

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountSearchBox.tsx [hunk 21]
```diff
@@ -56,27 +31,12 @@ const StyledInput = styled(Input)`
     }
 `;
```
Comment (dahaca) [inline packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountSearchBox.tsx:24]: old colors

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountItem.tsx [hunk 22]
```diff
@@ -25,29 +25,36 @@ interface WrapperProps {
...
     overflow: hidden;
-    padding-right: 10px;
-    margin-right: -10px;
+`;
+export const Right = styled.div`
+    overflow: hidden;
+    text-align: right;
 `;
 
 const Row = styled.div`
```
Comment (dahaca) [inline packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountItem.tsx:61]: old colors and typography in `AccountName`, `Balance` and `FiatValueWrapper`

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountGroup.tsx (hunk 19 above)
Comment (dahaca) [reply packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountGroup.tsx]: But also when you have CJ and default accs it's looks a bit off. I'd bring back "Default accounts" for when there are CJ accounts. Waiting for a final solution from Ondra.

image

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx [hunk 23]
```diff
@@ -34,6 +35,7 @@ const Wrapper = styled.div<{ isInline?: boolean }>`
 const MenuHeader = styled.div<{ isInline?: boolean }>`
     display: flex;
     flex-direction: column;
+    border-top: 1px solid ${({ theme }) => theme.borderOnElevation1};
```
Comment (dahaca) [inline packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx]: `borderOnElevation0` since `elevation1` is not visible in light mode.

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx (hunk 14 above)
Comment (dahaca) [reply packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx]: Maybe it's somehow related to the shadows?

File: packages/components/src/components/Dropdown/Dropdown.tsx (hunk 3 above)
Comment (dahaca) [reply packages/components/src/components/Dropdown/Dropdown.tsx:90]: Could you mention in in a [description](https://github.com/trezor/trezor-suite/blob/e955f00a4ecf3c045b50b61526c5a68fb9660864/packages/components/src/components/form/Select/Select.tsx#L182)?

File: packages/components/src/components/Dropdown/Dropdown.tsx [hunk 24]
```diff
@@ -87,7 +87,9 @@ export const Dropdown = forwardRef(
...
+            offsetX,
```
Comment (dahaca) [inline packages/components/src/components/Dropdown/Dropdown.tsx:91]: what are these for?

File: packages/components/src/components/Badge/Badge.tsx [hunk 25]
```diff
@@ -59,14 +59,24 @@ const getIconColor = (variant: BadgeVariant, isDisabled: boolean | undefined): C
     }
 };
 
+const getPadding = (size: BadgeSize) => {
+    switch (size) {
+        case 'tiny':
+            return `0 ${spacings.xs - spacings.xxxs}px`;
```
Comment (dahaca) [inline packages/components/src/components/Badge/Badge.tsx:65]: why not same as in `Button`?

File: packages/components/src/components/Badge/Badge.tsx (hunk 2 above)
Comment (dahaca) [reply packages/components/src/components/Badge/Badge.tsx:82]: you are right, thas was a placeholder style probably :/

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/TokensCount.tsx [hunk 26]
```diff
@@ -1,14 +1,11 @@
...
-import { spacingsPx } from '@trezor/theme';
+import { Badge } from '@trezor/components';
 
-const ButtonBadge = styled(Button)`
-    margin-left: ${spacingsPx.xs};
-    background: ${({ theme }) => theme.backgroundSurfaceElevation2};
-    color: ${({ theme }) => theme.textSubdued};
-    font-size: 10px;
-    padding: 1px 8px;
+const Container = styled.div`
```
Comment (dahaca) [inline packages/suite/src/components/wallet/WalletLayout/AccountsMenu/TokensCount.tsx]: how about `styled(Badge)` instead? Also, it should be round. 🙏

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx (hunk 14 above)
Comment (jvaclavik) [reply packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx]: Some of them can be deleted, but not these for shadows

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx (hunk 16 above)
Comment (jvaclavik) [reply packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx:113]: I need it now when I added CoinFilter there

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountItem.tsx (hunk 18 above)
Comment (jvaclavik) [reply packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountItem.tsx:38]: Yes, and only on hover. I felt the usability is better this way

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/TokensCount.tsx (hunk 26 above)
Comment (jvaclavik) [reply packages/suite/src/components/wallet/WalletLayout/AccountsMenu/TokensCount.tsx]: Btw I know why I don't like to overriding components from the design system. It's harder to refactor them if I'm looking for only for their usages in render 

Usually I can easily use this regexp: `<Button(.(?!>)|\n|\t)*size="small"` but I can't do it this way when I use override.

File: packages/suite/src/components/suite/AccountLabel.tsx (hunk 7 above)
Comment (jvaclavik) [reply packages/suite/src/components/suite/AccountLabel.tsx]: Yes, good idea!

File: packages/suite/src/components/suite/Preloader/SuiteLayout/Sidebar/Sidebar.tsx (hunk 8 above)
Comment (jvaclavik) [reply packages/suite/src/components/suite/Preloader/SuiteLayout/Sidebar/Sidebar.tsx:16]: There was some reason for it, probably borders to the edges, but not 100% sure. Quick actions fixed.

File: packages/components/src/components/Dropdown/Dropdown.tsx (hunk 24 above)
Comment (jvaclavik) [reply packages/components/src/components/Dropdown/Dropdown.tsx:91]: You can customize position of the dropdown. Example here:

Screenshot 2023-12-20 at 14 12 36

In the case of notification dropdown:

image

image

File: packages/components/src/components/Badge/Badge.tsx (hunk 25 above)
Comment (jvaclavik) [reply packages/components/src/components/Badge/Badge.tsx:65]: It's not circular with different values when we have one digit value there. And we need this for tokens in AccountsMenu. But we should definitely refactor it a bit.

File: packages/components/src/components/form/Input/Input.tsx (hunk 5 above)
Comment (jvaclavik) [reply packages/components/src/components/form/Input/Input.tsx]: I don't like using `className` with combination of styled-components, so I used `style` prop

File: packages/suite/src/components/suite/AccountLabel.tsx (hunk 7 above)
Comment (jvaclavik) [reply packages/suite/src/components/suite/AccountLabel.tsx]: Done

File: packages/suite/src/components/suite/Preloader/SuiteLayout/Sidebar/Sidebar.tsx (hunk 9 above)
Comment (jvaclavik) [reply packages/suite/src/components/suite/Preloader/SuiteLayout/Sidebar/Sidebar.tsx]: It's because of the borders, they should be to the edge of the sidebar, so I can't use `padding` in `Sidebar`

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountGroup.tsx (hunk 19 above)
Comment (jvaclavik) [reply packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountGroup.tsx]: We can move CJ accounts below default ones. 
I updated code a bit, but we will wait for the design decision. And if we are going to delete it we should delete also all translations for `TR_NORMAL_ACCOUNTS`

File: packages/suite/src/components/suite/Preloader/SuiteLayout/Sidebar/QuickActions.tsx [hunk 27]
```diff
@@ -30,15 +30,16 @@ const DesktopContainer = styled(WebContainer)`
...
     border-top: 1px solid ${({ theme }) => theme.borderOnElevation0};
 
     ::after {
         content: '';
         position: absolute;
         top: ${spacingsPx.xxs};
+        bottom: ${spacingsPx.xxs};
         right: 50%;
         width: 1px;
-        height: 100%;
```
Comment (jvaclavik) [inline packages/suite/src/components/suite/Preloader/SuiteLayout/Sidebar/QuickActions.tsx:41]: This created additional tiny scrolling for the whole sidebar which was not necessary

File: packages/components/src/components/Tooltip/Tooltip.tsx [hunk 28]
```diff
@@ -62,6 +62,8 @@ const Addon = styled.div`
 `;
 
 const Content = styled.div<{ dashed: boolean; cursor: Cursor }>`
+    cursor: ${({ cursor }) => cursor};
```
Comment (jvaclavik) [inline packages/components/src/components/Tooltip/Tooltip.tsx:65]: I'm not happy about this change but it didn't work properly for some elements like in AddAccountButton

File: packages/components/src/components/assets/CoinLogo/CoinLogoWithShareIndicator.tsx [hunk 29]
```diff
@@ -0,0 +1,114 @@
...
+                        strokeDashoffset: fillPercents,
+                    }}
+                    transition={transition}
+                />
+            </svg>
+        </>
+    );
+};
+
+export const CoinLogoWithShareIndicator = ({
```
Comment (dahaca) [inline packages/components/src/components/assets/CoinLogo/CoinLogoWithShareIndicator.tsx]: `AssetShareIndicator`? And can be outsode of the `CoinLogo` folder.

File: packages/suite/src/components/suite/AccountLabel.tsx (hunk 7 above)
Comment (dahaca) [reply packages/suite/src/components/suite/AccountLabel.tsx]: Add it [here](https://github.com/trezor/trezor-suite/blob/5049c83ac9b27ffad8875a8ea7d3814ee89f7b60/packages/suite/src/components/suite/AccountLabel.tsx#L27) so it affects user labels

File: packages/suite/src/components/suite/Preloader/SuiteLayout/Sidebar/Sidebar.tsx (hunk 9 above)
Comment (dahaca) [reply packages/suite/src/components/suite/Preloader/SuiteLayout/Sidebar/Sidebar.tsx]: I would add borders via `::after` to keep JSX clean, then you do not need to remove separate paddings or create `Margin` ;)

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/CoinsFilter.tsx (hunk 12 above)
Comment (dahaca) [reply packages/suite/src/components/wallet/WalletLayout/AccountsMenu/CoinsFilter.tsx]: Seems like those z-indicies I asked to remove were for the mobile menu somehow... looks like this now. And if you solve it via z-index, please leave a comment why it's there and preferable use [zIndices](https://github.com/trezor/trezor-suite/blob/67375b73905ddbba43dc8d1ec3784f626bcb5e51/packages/theme/src/zIndices.ts#L4).

image

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx (hunk 14 above)
Comment (dahaca) [reply packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx]: Please, leave comments for those so someone doesn't ask them to get deleted again in 3 years :D

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx [hunk 30]
```diff
@@ -88,18 +97,47 @@ const ExpandedMobileWrapper = styled.div`
...
     max-height: 80vh;
     overflow-y: auto;
     box-shadow: 0 4px 10px 0 ${({ theme }) => theme.BOX_SHADOW_BLACK_20};
     border-bottom-left-radius: 4px;
     border-bottom-right-radius: 4px;
-    padding: 0 16px;
     padding-bottom: 16px;
 `;
 
+const GradientBase = styled.div`
```
Comment (dahaca) [inline packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx]: nit: since the actual styled `div` is not needed, use this approach for when you have a base style and then simply spread it in `GradientBefore` and `After` ;)

```suggestion
const gradientBase = css`
```

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountGroup.tsx (hunk 19 above)
Comment (dahaca) [reply packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountGroup.tsx]: Moving is not the way, CJ is on the top on purpose.

Still waiting for an answer from Ondra but should not be anything major.

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/TokensCount.tsx (hunk 26 above)
Comment (dahaca) [reply packages/suite/src/components/wallet/WalletLayout/AccountsMenu/TokensCount.tsx]: Overriding obviously should not happen! You are right, dynamically changing border-radius when Badge has 1 character is cool as hell. And if you can create an issue or add it here it will definitely be nice. If not here, then a temporary override is better IMO. And sometimes you simply need to change margins or something, so `styled(Something)` is not always terrible.

File: packages/components/src/components/Tooltip/Tooltip.tsx (hunk 28 above)
Comment (dahaca) [reply packages/components/src/components/Tooltip/Tooltip.tsx:65]: then probably the one below is not needed? 🤔

File: packages/components/src/components/Dropdown/Dropdown.tsx (hunk 24 above)
Comment (dahaca) [reply packages/components/src/components/Dropdown/Dropdown.tsx:91]: Not sure it's needed since i believe it should be unified somehow. Why do you think extra effset should be applied in this case? I am afraid these props will be abused later.

File: packages/suite/src/components/suite/AccountLabel.tsx [hunk 31]
```diff
@@ -31,12 +32,15 @@ export const AccountLabel = ({
...
-            id="LABELING_ACCOUNT"
-            values={{
-                networkName: <Translation id={getTitleForNetwork(symbol)} />, // Bitcoin, Ethereum, ...
-                index: index + 1, // this is the number which shows after hash, e.g. Ethereum #3
-            }}
-        />
+        <TruncateWithTooltip>
+            <Translation
+                id="LABELING_ACCOUNT"
+                isNested
```
Comment (dahaca) [inline packages/suite/src/components/suite/AccountLabel.tsx:46]: wow, i didn't know this property exists even, honestly.

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx
Comment (dahaca) [inline packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx:1]: Shadow is now overlapping the account item. How about rendering them only when there is some scroll?

image

File: packages/components/src/components/Dropdown/Dropdown.tsx (hunk 24 above)
Comment (jvaclavik) [reply packages/components/src/components/Dropdown/Dropdown.tsx:91]: Maybe it's my subjective opinion and ADHD, but it looks a bit like optical illusion when dropdown background is white and button is gray. It looks like these shapes are not in one line. I think we should correct it to fake it (~ -2px or move it more to show clear intention).

image

In general I think it's pretty common use case correct these positions in dropdown. For me it's a bit similar like using override through styled-components. You want to add some padding so you are able to do it, buw why not here?

File: packages/components/src/components/Tooltip/Tooltip.tsx (hunk 28 above)
Comment (jvaclavik) [reply packages/components/src/components/Tooltip/Tooltip.tsx:65]: Maybe 🫣 but I can't assure it's working like that in all the places, so I used it this way.

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx
Comment (jvaclavik) [reply packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx:1]: Maybe yes. I wanted to avoid using javascript for it, but maybe it's not possible to do it if we want to have it correctly.

Comment (jvaclavik) [reply packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu.tsx:1]: Done

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/TokensCount.tsx (hunk 26 above)
Comment (jvaclavik) [reply packages/suite/src/components/wallet/WalletLayout/AccountsMenu/TokensCount.tsx]: It's not terrible, but it adds another complexity in my opinion. You never know if you are not overriding something inside of the component and searchability for refactoring is harder. Anyway I tried to use overriding here and it's not that easy in this case, so let's keep Container.

image

image

File: packages/components/src/components/Dropdown/Dropdown.tsx (hunk 24 above)
Comment (dahaca) [reply packages/components/src/components/Dropdown/Dropdown.tsx:91]: Yeah, I agree that it looks like an illusion :D I immediately want to come up with an algorithm and ratios to automate those offsets, so that they are relative to the toggle size or the dropdown size 😅  Because with the original toggle it does have an offset but it's less noticeable with a larget toggle, a `NavigationItem` in this case.

But yeah, leave the props! 🙌

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountGroup.tsx (hunk 19 above)
Comment (dahaca) [reply packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountGroup.tsx]: Created a follow-up [issue](https://github.com/trezor/trezor-suite/issues/10420).

File: packages/suite/src/components/wallet/WalletLayout/AccountsMenu/TokensCount.tsx (hunk 26 above)
Comment (dahaca) [reply packages/suite/src/components/wallet/WalletLayout/AccountsMenu/TokensCount.tsx]: I guess that's because `Badge` did not have a `className` prop? 🤔  Fixed 8ea1554da6

image
image

**Review by dahaca — APPROVED** [review]
